lib/init/grass.py

```diff
@@ -1804,11 +1841,11 @@ def show_info(shellname, grass_gui, default_gui):
         _("See citation options with:")))
 
     if grass_gui == 'wxpython':
-        message("%-41sg.gui wxpython" % _("If required, restart the GUI with:"))
+        message(f("%-41sg.gui wxpython", _("If required, restart the GUI with:")))
```

Comment:
f-strings are supported since Python 3.6. In requirements file (https://github.com/OSGeo/grass/blob/master/REQUIREMENTS.html#L58) we have Python >= 2.7 which needs apparently an update.

Reply:
From backport point of view (7.8), I would suggest to use `format()`

Reply:
Please take a closer look. This is not f-strings. It's the function `f()`. We need to hijack format strings and adjust string widths inside before `%` and `.format()`, so as far as I know, we cannot use `%` and `.format()` here. The `f()` function doesn't support named arguments yet.

Reply:
You are right, sorry for the noise.